CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.1 (2019-02-20)
+
+- Added `SkyTimepickerTimeOutput` to index barrel so it can be imported from the package root. [#17](https://github.com/blackbaud/skyux-datetime/pull/17).
```

Comment:
Can we use "Fixed" since this is a patch release?

Reply:
done.